integrations/http/save.go

```diff
@@ -55,33 +59,7 @@ func (h handler) handleAuth(w http.ResponseWriter, r *http.Request) {
 		auth = "Bearer " + bearer
 	}
 
-	// Save a new connection, and return to the user an autokitteh connection token.
-	connToken, err := h.createConnection(auth)
-	if err != nil {
-		l.Warn("Failed to save new connection secrets", zap.Error(err))
-		e := "Connection saving error: " + err.Error()
-		u := fmt.Sprintf("%serror.html?error=%s", uiPath, url.QueryEscape(e))
-		http.Redirect(w, r, u, http.StatusFound)
-		return
-	}
-
-	// Redirect the user to a success page: give them the connection token.
-	l.Debug("Saved new autokitteh connection")
-	u := fmt.Sprintf("%ssuccess.html?token=%s", uiPath, connToken)
-	http.Redirect(w, r, u, http.StatusFound)
-}
+	initData := sdktypes.NewVars().Set(authVar, auth, true)
```

Comment:
Please add a comment (store connection vars, redirect the users, etc.)

Reply:
added, though i think it's a bit redundant.